Flick/Views/ModalView.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ModalView.swift
+//  Flick
+//
+//  Created by Lucy Xu on 6/20/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import UIKit
+
+class ModalView: UIView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
what's this file for?

Reply:
its for my pop up haha.. its from a branch im working on right now.